core/io/resource_format_binary.cpp

```diff
@@ -693,7 +693,7 @@ Error ResourceLoaderBinary::load() {
 
 			if (external_resources[i].cache.is_null()) {
 				if (!ResourceLoader::get_abort_on_missing_resources()) {
-					ResourceLoader::notify_dependency_error(local_path, path, external_resources[i].type);
+					callable_mp_static(&ResourceLoader::notify_dependency_error).bind(local_path, path, external_resources[i].type).call_deferred();
```

Comment:
I think I would implement the deferred call in the editor side on whathever handles this static function for notifying dependency errors, the loader in itself is kind of agnostic to threading.

Reply:
If neither `ResourceLoader` implementations nor the editor have to be aware of threading, the only option remaining is that the static function `ResourceLoader::&ResourceLoader::notify_dependency_error()` be the one that "bridges" between the multi-threaded loader world and the single-threaded editor world. In short, that function would be the one deferring the call to the registered notify function (which happens to be editor's).
Would that make sense?

Reply:
In case it does, I've already done the change, so please re-review.